egui/src/widgets/plot.rs

```diff
@@ -440,17 +491,16 @@ impl Widget for Plot {
             bounds.max[1] = y_abs;
         };
 
-        bounds.expand_x(margin_fraction.x as f64 * bounds.width());
-        bounds.expand_y(margin_fraction.y as f64 * bounds.height());
-
         if let Some(data_aspect) = data_aspect {
             let data_aspect = data_aspect as f64;
             let rw = rect.width() as f64;
             let rh = rect.height() as f64;
             let current_data_aspect = (bounds.width() / rw) / (bounds.height() / rh);
-            if current_data_aspect < data_aspect {
+
+            let margin = 1e-5;
```

Comment:
A more common name for this is `epsilon` (something used to avoid rounding errors)

Reply:
Fixed in 1b00e7f